topics/data-management/metadata.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: data-management
 type: data-science
-enable: false
+#enable: false
```

Comment:
So this'll make it visible on the homepage and I'm assuming y'all want to put up more content first and maybe do a bit more of the stylistic changes. But I think I have a better solution! 

Until now "enable" meant really don't show it to anyone, but we know when we're doing a 'production' build, and we can make enable: false only hide it when galaxyproject/training-material is deployed, rather than when everyone else deploys it.

I'll make that change and then that can be a bit easier for y'all maybe?